wagtail/images/views/bulk_actions/delete.py

```diff
@@ -0,0 +1,37 @@
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext
+
+from wagtail.core import hooks
+from wagtail.images.views.bulk_actions.image_bulk_action import ImageBulkAction
+
+
+class DeleteBulkAction(ImageBulkAction):
+    display_name = _("Delete")
+    action_type = "delete"
+    aria_label = _("Delete images")
+    template_name = "wagtailimages/bulk_actions/confirm_bulk_delete.html"
+    action_priority = 10
```

Comment:
suggestion: move this last (so maybe something like action_prioty 100) since it is destructive and it does look odd as the first item in the list